web/pages/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import classNames from "classnames";
+
+import styles from "./Layout.module.css";
+
+type LayoutProps = {
+  children: React.ReactNode;
+};
+
+const links: { label: string; link: string }[] = [
+  { label: "Home", link: "/" },
+  { label: "Podcast", link: "/podfeed" },
+  { label: "Lines", link: "/lines" },
+  { label: "Game Grid", link: "/game-grid" },
+  { label: "Stats", link: "/stats" },
+  { label: "Charts", link: "/charts" },
+  { label: "Blog", link: "/blog" },
+];
+
+function Layout({ children }: LayoutProps) {
+  const router = useRouter();
+  console.log(router.pathname);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.navBarLeftSide}>
+        <nav>
+          <ul>
+            {/* Links in Nav Bar ("currentPage" on whichever page you're on) -----------*/}
```

Comment:
These links are just placeholder. Write some comments to denote that.

Reply:
ok